server/routes/authRoute.js

```diff
@@ -40,21 +40,12 @@ router.post(
 		try {
 			/* See if user exists */
 			let user = await User.findOne({ email });
-
-			if (!user) {
-				return res
-					.status(400)
-					.json({ errors: 'Invalid Credentials'});
-			}
+			if (!user) return res.json({ error: 'Invalid Credentials'});
```

Comment:
Got rid of status code because of error handling.

Reply:
Should we keep 400 status?

Reply:
Keep this one for now. I need it for testing.